app/frontend/Mail/MailChat/MailMessage.svelte

```diff
@@ -2,7 +2,7 @@
   <svelte:fragment slot="above-center">
     <hbox flex />
     <hbox class="recipients">
-      to: {recipientsLine}
+      {$t`to:`} {recipientsLine}
```

Comment:
```suggestion
      {$t`to:`}
      {recipientsLine}
```

It's correct that the recipientsLine is not part of the string.

Let's add a newline to make clear that these are 2 different elements.